OpenRA.Mods.Common/Widgets/ProductionPaletteWidget.cs

```diff
@@ -64,6 +64,8 @@ public enum ReadyTextStyleOptions { Solid, AlternatingColor, Blinking }
 		public readonly string NotBuildableSequence = "idle";
 		public readonly string NotBuildablePalette = "chrome";
 
+		public readonly bool DrawTime = true;
```

Comment:
It might make sense to expose multiple flags, or have it as an enum, allowing control of all three label types?

Reply:
it doesnt make much sense for the build / queue amount, as there is no other way to display that information. the other strings can simply be set to emptystring, so no special function is required.